### DISCUSSION TITLE
Author of the commit

### DISCUSSION THREAD

**Discussion by msvitok77** [NeogitOrg/neogit#477] [Q&A] [answered]
Hello,
when I do :Neogit log<cr> 
then press `l`
I get something like this:
![image](https://user-images.githubusercontent.com/13028856/225951713-db6673c9-704f-4b07-9889-e4561407a855.png)
Is it possible to configure it, to show the author of that commit?

Thanks.

**Comment by msvitok77** [comment, marked as answer]
I hacked it myself. Just add these lines:

```
      text(" "),
      text("➜ "),
      text(commit.author_name, { highlight = "Character" }),
      text(" <"),
      text(commit.author_date, { highlight = "Number" }),
      text(">"),

```

here:
.local/share/nvim/site/pack/packer/start/neogit/lua/neogit/buffers/log_view/ui.lua
function: Component.new

**Reply by msvitok77** [reply]
be aware that these changes get removed after you update this plugin using packer ;-).

**Reply by CKolkey** [reply]
Have you tried hitting `<tab>` with your cursor on a commit? That said, I like showing more info up front :)

**Reply by msvitok77** [reply]
sure but I want to have it displayed from the start. I know it's possible to view the details by hitting tab or by hitting `<c-k> <c-j>`, but as I said I want to have all those details from the start. 🤷🏻‍♂️

**Reply by CKolkey** [reply]
Totally fair.

**Reply by msvitok77** [reply]
config option would be great. If I have time - I'll look at it.
